src/celeritas/ext/detail/CelerEmStandardPhysics.cc

```diff
@@ -435,6 +461,67 @@ void CelerEmStandardPhysics::add_e_processes(G4ParticleDefinition* p)
     }
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Add EM processes for muons.
+ *
+ * | Processes                    | Model classes                |
+ * | ---------------------------- | ---------------------------- |
+ * | Pair production              | G4MuPairProductionModel      |
+ * | Ionization (low E, mu-)      | G4ICRU73QOModel              |
+ * | Ionization (low E, mu+)      | G4BraggModel                 |
+ * | Ionization (high E)          | G4MuBetheBlochModel          |
+ * | Bremsstrahlung               | G4MuBremsstrahlungModel      |
+ * | Coulomb scattering           | G4eCoulombScatteringModel    |
+ * | Multiple scattering          | G4WentzelVIModel             |
+ *
+ * \note Currently all muon processes are disabled by default
+ */
+void CelerEmStandardPhysics::add_mu_processes(G4ParticleDefinition* p)
+{
+    auto* physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
+
+    if (options_.mu_pair_production)
+    {
+        physics_list->RegisterProcess(new G4MuPairProduction(), p);
+        CELER_LOG(debug) << "Loaded muon pair production with "
+                            "G4MuPairProductionModel";
+    }
+
+    if (options_.mu_ionization)
+    {
+        physics_list->RegisterProcess(new G4MuIonisation(), p);
+        CELER_LOG(debug) << "Loaded muon ionization with G4ICRU73QOModel, "
+                            "G4BraggModel, and G4MuBetheBlochModel";
+    }
+
+    if (options_.mu_bremsstrahlung)
+    {
+        physics_list->RegisterProcess(new G4MuBremsstrahlung(), p);
+        CELER_LOG(debug) << "Loaded muon bremsstrahlung with "
+                            "G4MuBremsstrahlungModel";
+    }
+
+    if (options_.mu_coulomb)
+    {
+        // TODO: update the Celeritas Coulomb scattering model to support muons
```

Comment:
If these aren't todos that aren't next on the list to be updated, could you annotate them as doxygen so that they automatically show up on the [todo list](https://celeritas-project.github.io/celeritas/dev/dd/da0/todo.html)?
```suggestion
        //! \todo Update the Celeritas Coulomb scattering model to support muons
```

Reply:
Oh nice, didn't realize they got put in a list.